src/parser/parsers/ability_cards.py

```diff
@@ -50,6 +50,12 @@ def run(self):
                     if parsed_ui is not None:
                         hero_abilities[self.ability_index] = parsed_ui
                 except Exception as e:
+                    # TEMPORARY, IDK HOW TO GET AROUND IT PERMANENTLY
+                    if ability['Key'] in [
+                        'ability_nano_proximity_ritual',
+                        'ability_viper_snakedash',
+                    ]:
+                        continue
```

Comment:
no errors on my end, nor on the github actions, so must be either your environment, or your code changes